### DISCUSSION TITLE
Moordyn and map

### DISCUSSION THREAD

**Discussion by 3sdb** [OpenFAST/openfast#2004] [General]
Everyone
I have a question why does this error occur?
![141c668251eaf81bed4fa6c4a386d0fa](https://github.com/OpenFAST/openfast/assets/156734919/31911a71-06de-4ddd-9e89-b887944c1578)
![382d06bbf379dcaa3a8205814823063f](https://github.com/OpenFAST/openfast/assets/156734919/37a7e4ab-dfd4-424f-9f3e-ad8df2f5c7cb)
![fb761150589dc69d8db9ae61541abc68](https://github.com/OpenFAST/openfast/assets/156734919/0372ee03-9b06-454d-a6ef-4d50a0e975ef)

**Comment by jjonkman** [comment]
Dear @3sdb,

The problem is that you are calling MAP++, but are using a MoorDyn input file. When using MAP++, you'll need to use an input file formatted for MAP++.  For the MIT/NREL TLP, a MAP++ input file is available here: https://github.com/OpenFAST/r-test/blob/main/glue-codes/openfast/5MW_TLP_DLL_WTurb_WavesIrr_WavesMulti/NRELOffshrBsline5MW_MIT_NREL_TLP_MAP.dat.

Best regards,
